src/test/js/karaoke-noStages.js

```diff
@@ -26,15 +26,19 @@ module.exports = {
         browser.waitForElementVisible('code')
             .click('code')
             .keys(browser.Keys.UP_ARROW)
-            .getText('code', function (result) {
-                const text = result.value;
-                this.pause(1000)
-                    .waitForElementVisible('code')
-                    .getText('code', function (result) {
-                        this.assert.equal(text, result.value)
-                    });
-
-            });
+            .elements('css selector', 'div.result-item', function (resutlItems) {
+                var results = resutlItems.value.length;
+                // to validate that we left follow, give it some time and then count the elements again
+                this.pause(3000)
+                    .elements('css selector', 'code', function (codeCollection) {
+                        // JENKINS-36700 there can only be one code view open in follow stopped
+                        this.assert.equal(codeCollection.value.length, 1);
+                    })
+                    .elements('css selector', 'div.result-item', function (resutlItemsCompare) {
+                        // there should not be more items then we had before
+                        this.assert.equal( resutlItemsCompare.value.length, results);
+                    })
+                });
```

Comment:
@scherler isn't there a way to just tick the timer instead of an actual pause? (or maybe is that what pause is actually doing?)